packages/wrangler/templates/pages-dev-pipeline.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ *
+ */
+
+// @ts-ignore entry point will get replaced
+import worker from "__ENTRY_POINT__";
+// @ts-ignore entry point will get replaced
+export * from "__ENTRY_POINT__";
+
+const transformToRegex = (filter: string) => {
+	return filter.replace("*", ".*");
+};
+
+const routes = {
+	// @ts-ignore routes are injected
```

Comment:
instead of using `@ts-ignore` we can be a bit type-safer and do something like this:

```tsx
declare const __ROUTES__: unknown

if (!isRoutesJSONSpec(__ROUTES__)) {
  throw new Error('Missing global __ROUTES__ value')
}
```

Reply:
✅

Reply:
@GregBrimble where would we import `isRoutesJSONSpec` from tho? The above fails with 
```
✘ [ERROR] [plugin alias] unrecognized module: ./../src/pages/functions/routes-validation
```
This must be one of those esbuild things I don't fully get yet

Reply:
Not easily done here. This template runs in the user-space, whereas that `src/pages` stuff is compiled into wrangler. I think it's good enough as you've got it now :)